package.json

```diff
@@ -41,10 +41,10 @@
   },
   "homepage": "https://github.com/fastify/fastify-swagger#readme",
   "devDependencies": {
+    "@fastify/basic-auth": "^3.0.0",
+    "@fastify/helmet": "^8.0.0",
```

Comment:
this new versions includes the helment 5 and drops node 10
https://github.com/fastify/fastify-helmet/releases/tag/v6.0.0

I think it is fine to keep the v5 till fastify v4

Reply:
Cool beans, will revisit with v4, @fastify-static has already been updated in another PR.